sdk/device.go

```diff
@@ -130,13 +130,15 @@ type Device struct {
 	SortOrdinal int32
 }
 
+// TODO: Should be Json casing not JSON.
+// (golang -> PascalCasing is public, camelCasing is private)
```

Comment:
my linter tells me it should be `JSON` not `Json`

its not so much that its PascalCasing vs camelCasing which determines public v private in golang, its just whether the name is capitalized or not

Reply:
I'll relent on this. It's inconsistent.